django/test/client.py

```diff
@@ -38,7 +38,7 @@
 
 BOUNDARY = 'BoUnDaRyStRiNg'
 MULTIPART_CONTENT = 'multipart/form-data; boundary=%s' % BOUNDARY
-CONTENT_TYPE_RE = _lazy_re_compile(r'.*; charset=([\w\d-]+);?')
+CONTENT_TYPE_RE = _lazy_re_compile(r'.*; charset=([\w-]+);?')
```

Comment:
This is because `\w` already includes `\d`; I wonder if we should switch that to `[a-zA-Z0-9_-]` though since charsets would probably be ASCII, can be done in another commit though

Reply:
> This is because `\w` already includes `\d`;

Yes, exactly.

> I wonder if we should switch that to `[a-zA-Z0-9_-]` though since charsets would probably be ASCII, can be done in another commit though

Yes, can be done as a separate cleanup.